spec.md

```diff
@@ -1495,6 +1626,112 @@ For success responses, the following fields are defined:
 }
 ```
 
+## Adapting Service Binding Credentials
+
+If the endpoints of a Service Instance are from an Application point of view
+different to the endpoints provided in the binding credentials, then the
+binding credentials MUST be adapted before the Platform provides them to
+the Application.
+
+The Platform requests the corrected credentials from the Service Broker by
+providing the original credentials and a table that maps the original
+endpoints to the new endpoints.
```

Comment:
The platform itself is not necessarily always able to create this table. Depending on the underlying transport mechanism, sometimes the platform is only able to control the correct target hostname, while the service broker controls the correct target port. In these cases, the platform would be unable to request an adaptation as specified here, since it would be unable to create a valid mapping table.